fusionauth/testdata/messages.properties.go

```diff
@@ -10,7 +10,7 @@ func MessageProperties(name string) string {
 
 	return fmt.Sprintf(`
 #
-# Copyright (c) 2019-2021, FusionAuth, All Rights Reserved
```

Comment:
Based on update of messages from https://github.com/gpsinsight/terraform-provider-fusionauth/pull/216